docs/installation/vue3.md

```diff
@@ -106,6 +106,26 @@ export default {
 </script>
 ```
 
+Or feel free to use the new [`<script setup>` syntax](https://v3.vuejs.org/api/sfc-script-setup.html).
+
+```html
```

Comment:
GitHub does support `vue` as a possible code block language but unsure/haven't tested if the documentation generator here will support it.

Using `html` like other code blocks in the mean time